src/libraries/System.Runtime/tests/System.Globalization.Tests/CultureInfo/CultureInfoCurrentCulture.cs

```diff
@@ -42,6 +42,10 @@ public void CurrentCulture_Default_Not_Invariant()
             // culture.
             Assert.NotEqual(CultureInfo.CurrentCulture, CultureInfo.InvariantCulture);
             Assert.NotEqual(CultureInfo.CurrentUICulture, CultureInfo.InvariantCulture);
+            
+            // The current culture taken from default system culture should be specific
+            // TODO: https://github.com/dotnet/runtime/issues/111501
+            //Assert.False(CultureInfo.CurrentCulture.IsNeutralCulture);
```

Comment:
I would suggest instead of adding this as a comment, to make a separate test method with a `Skip` attribute specifying the reason for exception i.e. GH issue you created.